Snippets/Snippets_6/DataBus/CleanupStrategy/Define.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Snippets6.DataBus.CleanupStrategy
+{
+    using NServiceBus;
+
+    public class Define
+    {
+        public Define()
+        {
+            #region DefineFileLocationForDatabusFiles
+            EndpointConfiguration endpointConfiguration = new EndpointConfiguration("endpointName");
+            endpointConfiguration.UseDataBus<FileShareDataBus>().BasePath(@"c:\database_files\");
```

Comment:
Should this be databus_files or databus_attachments? instead of `\database_files\`?